### PR TITLE
feat: add drop offline table api and update drop table in CLI

### LINKED ISSUE #858: Dropping tables will delete offline data as well
Now dropping data will not delete offline data which generates unmanaged data.

We should delete the offline data if it is deep-copied and remove meta data if it is symbol-import.

### LINKED ISSUE #961: Add drop offline table for TaskManager and BatchJob API
Add the new api to drop offline table.

### PR DESCRIPTION
**Pull request by tobegit3hub**
* Resolve https://github.com/4paradigm/OpenMLDB/issues/858 .
* Add new api for TaskManager to drop offline table
* Update drop table command in CLI

### CODE REVIEW

**Review by vagetablechicken — APPROVED** [review]

File: src/sdk/sql_router.h [hunk 1]
```diff
@@ -76,6 +76,8 @@ class SQLRouter {
 
     virtual bool DropDB(const std::string& db, hybridse::sdk::Status* status) = 0;
 
+    virtual bool DropTable(const std::string& db, const std::string& table, hybridse::sdk::Status* status) = 0;
```
Comment (dl239) [inline src/sdk/sql_router.h]: why add `DropTable` interface? We can execute drop table cmd by `ExecuteDDL`

Comment (tobegit3hub) [reply src/sdk/sql_router.h]: I will remove this if we have supported `drop table` in SQL.

**Comment by dl239** [comment]
need to call the added `DropTable` function here https://github.com/4paradigm/OpenMLDB/blob/main/src/sdk/sql_cluster_router.cc#L1177

**Comment by tobegit3hub** [comment]
> need to call the added `DropTable` function here https://github.com/4paradigm/OpenMLDB/blob/main/src/sdk/sql_cluster_router.cc#L1177

Updated.

**Review by dl239 — APPROVED** [review]

**Comment by tobegit3hub** [comment]
[CLI for offline jobs with unified metadata store](https://trello.com/c/rcSLPVcy/21-cli-for-offline-jobs-with-unified-metadata-store)

**Pull request merged**